inbc-program/inbc/parser/parser.py

```diff
@@ -65,11 +65,11 @@ def parse_source_args(self) -> None:
 
         # Application Add Command
         app_add_parser = app_subparsers.add_parser('add')
-        app_add_parser.add_argument('--gpgKeyUri', '-gku', required=True,
+        app_add_parser.add_argument('--gpgKeyUri', '-gku', required=False,
                                     type=lambda x: validate_string_less_than_n_characters(
                                         x, 'str', 1500),
                                     help='Uri from which to download GPG key')
-        app_add_parser.add_argument('--gpgKeyName', '-gkn', required=True,
+        app_add_parser.add_argument('--gpgKeyName', '-gkn', required=False,
                                     type=lambda x: validate_string_less_than_n_characters(
                                         x, 'str', 200),
                                     help='Name to store the GPG key information')
```

Comment:
**Suggestion:** The '--gpgKeyUri' and '--gpgKeyName' arguments have been changed from required to optional. Ensure that the rest of the codebase, especially the 'application_add' function, is updated to handle cases where these arguments might not be provided. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        # No change to the code itself, but ensure compatibility with the rest of the codebase.
```